docs/integrations-guides/rollup-guides/orbit/deployment.md

```diff
@@ -21,53 +21,68 @@ Arbitrum nodes communicate with EigenDA via the proxy for secure communication a
 
 1. Assuming you have yarn and hardhat installed. 
 
-2. Download the nitro contracts source [code](https://github.com/Layr-Labs/nitro-contracts) from the EigenDA fork using latest stable version.
+2. Download the nitro contracts source [code](https://github.com/Layr-Labs/nitro-contracts) from the EigenDA Nitro contracts fork using a latest stable version [release](https://github.com/Layr-Labs/nitro-contracts/releases).
```

Comment:
> using a latest stable version

nit: should probably either be "using *the* latest stable version", *or* "using a ~~latest~~ stable version"

Reply:
0cbc508970a0d7f6d09f81618ff6f2fe544bcbf9